pinot-common/src/main/java/org/apache/pinot/pql/parsers/Pql2Compiler.java

```diff
@@ -72,7 +72,8 @@
       Boolean.valueOf(System.getProperty("pinot.query.converter.fail_on_error", "false"));
   public static String ENABLE_DISTINCT_KEY = "pinot.distinct.enabled";
   public static boolean ENABLE_DISTINCT = Boolean.valueOf(System.getProperty(ENABLE_DISTINCT_KEY, "true"));
-
+  public static String ENABLE_TEXT_MATCH_KEY = "pinot.textmatch.enabled";
```

Comment:
why is this needed?

Reply:
TEXT index is currently disabled (temporarily) until support for this is added in segment reload path. The feature will be enabled and this check will be removed in the immediate follow-up PR when we add changes for segment reload.

A similar check is there in segment generator code path to prevent usage of this feature on existing tables